### PR TITLE
fix OCP-22606

### PR DESCRIPTION
**Pull request by sandeepknd**
fix OCP-22606

Hi @geliu2016 
Could you please have a look. Excluded this test from execution on disconnected profiles.

### CODE REVIEW

**Comment by geliu2016** [comment]
@sandeepknd this etcd operator: etcd-operator-alm-owned is not cluster's etcd-operator, this is olm managed service provided to customer to save their own data if necessary, there is subscription file to create this resource. so there is two different operators:
A: etcd-operator, located in project: openshift-etcd-operators, and we can't touch it.
B: etcd-operator-alm-owned, located in project: openshift-operators, this is customerized resource which we created with subscription.yaml file

**Comment by sandeepknd** [comment]
Hi @geliu2016 ,
Excluded this test from execution on disconnected profiles.

File: features/upgrade/etcd/upgrade.feature [hunk 1]
```diff
@@ -38,7 +38,7 @@ Feature: basic verification for upgrade testing
   @4.11 @4.10 @4.9 @4.8 @4.7 @4.6
   @vsphere-ipi @openstack-ipi @gcp-ipi @baremetal-ipi @azure-ipi @aws-ipi
   @vsphere-upi @openstack-upi @gcp-upi @baremetal-upi @azure-upi @aws-upi
-  @proxy @noproxy @disconnected @connected
+  @proxy @noproxy @connected
```
Comment (liangxia) [inline features/upgrade/etcd/upgrade.feature:41]: You need to rebase your PR and remove the same tag from prepare step, too.

Comment (sandeepknd) [reply features/upgrade/etcd/upgrade.feature:41]: done. 
@liangxia ,
Could you please have a look.

**Pull request merged**